google-cloud-firestore-v1/.owlbot.rb

```diff
@@ -0,0 +1,25 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Fixes invalid request headers.
+# See internal issue b/274072959.
```

Comment:
Since the internal issue referenced is a mirrored copy of the external GitHub issue, I suggest using that instead to avoid using unnecessary internal references: https://github.com/googleapis/google-cloud-ruby/issues/20926

Reply:
Unless there's a reason the internal issue link is preferred?

Reply:
The current fix is a temporary solution and should be removed once the gapic generator is updated with the new template. A parent issue has been created to track when the patch added in this PR should be removed.